juneau-core/src/main/java/org/apache/juneau/internal/StringUtils.java

```diff
@@ -618,7 +618,9 @@ public static boolean isEmpty(String s) {
 	 * @return <jk>true</jk> if specified string is <jk>null</jk> or it's {@link #toString()} method returns an empty string.
 	 */
 	public static boolean isEmpty(Object s) {
-		return s == null || s.toString().isEmpty();
+		if( s == null ) return true;
+		if( s instanceof List ) return ((List) s).size() == 0;

```

Comment:
There is a big discussion on stackoverflow on this topic. 
https://stackoverflow.com/questions/3321526/should-i-use-string-isempty-or-equalsstring
I like the simple 
"".equals(s)
which handles the null case.

If you include s being an instance of collection, it seems that you could use
if (s instanceof Iterable) return (Iterable) s).isEmpty();
Then even user-defined types that implement Iterable can be used.